### PR TITLE
add meta tags for social link display

### PR DESCRIPTION
**Pull request by elvece**
These meta tags conform to the [standard](https://ogp.me/) for sharing data on social media sites. 
Feel free to leave feedback with preferred text - I used my best judgement.

### CODE REVIEW

File: _includes/meta.html [hunk 1]
```diff
@@ -1,3 +1,8 @@
 <meta content='text/html; charset=utf-8' http-equiv='Content-Type'>
 <meta http-equiv='X-UA-Compatible' content='IE=edge'>
 <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no">
+<meta property="og:title" content="Grin - The Mimblewimble Blockchain">
+<meta property="og:description" content="A privacy-preserving digital currency with no amounts or addresses. Electronic transactions for all. Without censorship or restrictions. Grin is designed for the decades to come, not just tomorrow. Grin wants to be usable by everyone — regardless of borders, culture, skills or access.">
+<meta property="og:image" content="https://grin.mw/assets/images/img1@3x.png">
+<meta property="og:url" content="https://grin.mw/">
+<meta name="twitter:card" content="summary_large_image">
```
Comment (quentinlesceller) [inline _includes/meta.html:8]: Do we really want this Twitter meta? If yes, should we add the `twitter:title` property as it seems required here https://developer.twitter.com/en/docs/tweets/optimize-with-cards/overview/summary-card-with-large-image ? Usure if the `og:title` is enough in that case.

Comment (elvece) [reply _includes/meta.html:8]: @quentinlesceller Yes. Twitter will default to using `og:title` in the absence of `twitter:title`. Unless there should be a distinct title for Twitter and in the interest of consolidation, I think this approach is suitable. I am going to change the `twitter:card` content to "summary" instead, as this is more standard.

Comment (quentinlesceller) [reply _includes/meta.html]: Thanks you for clarifying !

**Comment by quentinlesceller** [comment]
Thank you for the PR @elvece 🎆. LGTM 🚀

**Pull request merged**